src/test/unit/motor_output_unittest.cc

```diff
@@ -28,6 +28,41 @@ extern "C" {
 #include "unittest_macros.h"
 #include "gtest/gtest.h"
 
+extern "C" {
+	uint8_t atomic_BASEPRI;
+
+	bool featureIsEnabled(uint8_t f);
+	float getDigitalIdleOffset(void *cfg);
+	float scaleRangef(float a, float b, float c, float d, float e);
+
+}
+
+// Mocking functions
+
+bool featureIsEnabled(uint8_t f)
+{
+	(void)f;
+	return true;
+}
+
+float getDigitalIdleOffset(void *cfg)
+{
+	(void)cfg;
```

Comment:
```suggestion
	UNUSED(cfg);
```

Reply:
Or directly: `float getDigitalIdleOffset(void *)`

Reply:
fixed

Reply:
But now the stub doesn't perfectly represent the original function anymore and you have to look up what actually goes into the function. IMO it would be better to use `const motorConfig_t *` here and then explicitly cast to `void *` in the stub, so that the function header is what it is supposed to be.

Reply:
fixed